ion/src/device/shared/usb/dfu_interface.h

```diff
@@ -70,11 +54,10 @@ class DFUInterface : public Interface {
     GetStatus    = 3,
     ClearStatus  = 4,
     GetState     = 5,
-    Abort        = 6,
-    Unlock       = 11
+    Abort        = 6
   };
 
-  // DFU Download Command Codes
+  // DFU Download Commmand Codes
```

Comment:
Why do you edit it ?
```suggestion
  // DFU Download Command Codes
```